examples/modify_timestep.rs

```diff
@@ -112,8 +106,8 @@ fn mouseclick(
     if mouse_input.just_pressed(MouseButton::Left) {
         let duration = step.get_duration();
         step.set_duration(*other_duration);
-        text.single_mut().sections[0].value = format!(
-            "Spatial Update Rate: every {}ms",
+        text.single_mut().0 = format!(
+            "Spatial Update Rate: {} ms",
```

Comment:
This was weird, I changed the string text slightly here because it appears the length of the old text was triggering a bug in Bevy 0.15 and caused the end of the text to not render. Just a heads up in case you see anything similar.

Will try to file a bug report with Bevy once I can confirm there isn't one already and make a reproduction.